### ISSUE TITLE
configure backend for jacfwd  call

### ISSUE THREAD

**Issue by mganahl** [google/jax#5535]
Hi! Just wanted to quickly ask if it's possible to configure the xla backend for calls to `jacfwd`. As part of some larger calculation on TPUs we're doing some preprocessing which involves computing gradients of ops acting on complex arrays. Currently the `jacfwd` call just hangs (I assume it's related to the complex dtypes involved).

Thanks in advance!

**Comment by mganahl** [comment]
seems like this is an issue with tpu driver, closing this.

**Comment by mganahl** [comment]
Just coming back to this with a question: What is the current status with the `backend` argument to `jit`? We're trying to do some data-preprocessing on host VMs connected to a TPU, but would like to use the host CPU to perform the preprocessing (which involves `jax.jit` and `jax.jacfwd`) in double precision arithmetic. Passing `backend=cpu` doesn't seem to do the trick.

**Comment by skye** [comment]
Hi, sorry for the delayed reply. Per your original question, complex numbers aren't currently supported with the TPU driver. This will be addressed with the new Cloud TPU architecture we announced late last year (see this [tweet](https://twitter.com/jekbradbury/status/1337528357517291520) for video + slides). 

`backend=cpu` should work as of https://github.com/google/jax/pull/5751. I suggest installing jax from github head to pick this up:
```
git clone https://github.com/google/jax
cd jax
pip install --upgrade -e .
```

**Comment by mganahl** [comment]
@skye thanks for the answer and the fix! Will test it asap!
